src/components/tooltip/tooltip.scss

```diff
@@ -19,4 +19,27 @@ $md-tooltip-padding: 8px;
   margin: $md-tooltip-margin;
   height: $md-tooltip-height;
   line-height: $md-tooltip-height;
+  opacity: 0;
+  pointer-events: none;
+  transform: scale(0);
+  &.md-initialized {
```

Comment:
Rather than creating these as nested classes, can they each be a standalone `md-tooltip-above`, etc.?